src/compiler/expand_fcompute.cc

```diff
@@ -0,0 +1,48 @@
+/*!
+ * Copyright (c) 2018 by Contributors
+ * \file expand_fcompute.cc
+ * \author Nick Hynes
+*/
+#include <nnvm/graph.h>
+#include <nnvm/op_attr_types.h>
+#include <nnvm/graph_attr_types.h>
+#include <nnvm/pass.h>
+#include <nnvm/compiler/op_attr_types.h>
+#include <nnvm/top/nn.h>
+#include "./graph_transform.h"
+#include "./pattern_util.h"
+
+namespace nnvm {
+namespace compiler {
+
+Graph ExpandCompute(nnvm::Graph src) {
+  const IndexedGraph& idx = src.indexed_graph();
+  std::unordered_map<std::string, uint32_t> name2idx;
```

Comment:
this bit is required when invoking `ftransform` on a node's inputs requires a copy. being a copy, the input pointer doesn't exist in the indexed graph and can't be fetched using `idx.entry_id`.